modules/cpmso-yaml-provider-spec-gcp.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-provider-spec-gcp_{context}"]
+= Sample GCP provider specification
+
+When you create a control plane machine set for an existing cluster, the provider specification must match the `providerSpec` configuration in the control plane `machine` CR that is created by the installation program. You can omit any field that is set in the failure domain section of the CR.
+
+In the following example, `<cluster_id>` is the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI installed, you can obtain the infrastructure ID by running the following command:
+
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+
+.Sample GCP `providerSpec` values
```

Comment:
This is taken from a cluster today, and scrubbed to make it generic. I have some questions below about specific items, and the whole thing needs verification.

Reply:
Number 1: This is not relevant IMO.
Number 3: This is also not relevant. Says whether the resource should be protected against deletion.
Number 4: Image is the source image to create this disk.